tempesta_fw/http.c

```diff
@@ -160,35 +123,57 @@ tfw_http_prep_hexstring(char *buf, u_char *value, size_t len)
 	}
 }
 
+/*
+ * HTTP 302 response.
+ * The response redirects the client to the same URI as the original request,
+ * but it includes 'Set-Cookie:' header field that sets Tempesta sticky cookie.
+ */
 TfwHttpMsg *
-tfw_http_prep_302(TfwHttpMsg *hm, TfwStr *cookie)
+tfw_http_prep_302(TfwHttpMsg *hmreq, TfwStr *cookie)
 {
+#define S_302_PART_01	S_302 S_CRLF S_F_DATE
+#define S_302_PART_02	S_CRLF S_F_CONTENT_LENGTH "0" S_CRLF S_F_LOCATION
+#define S_302_PART_03	S_CRLF S_F_SET_COOKIE
+#define S_302_FIXLEN	SLEN(S_302_PART_01 S_V_DATE S_302_PART_02 S_302_PART_03)
+#define S_302_KEEP	S_CRLF S_H_CONN_KA
+#define S_302_CLOSE	S_CRLF S_H_CONN_CLOSE
```

Comment:
Please don't place so many defines inside the function, it's good to place them bit ahead of the function.